packages/services/src/kernel/messages.ts

```diff
@@ -165,6 +165,22 @@ export namespace KernelMessage {
     return msg.header.msg_type === 'update_display_data';
   }
 
+  /**
+   * Test whether a kernel message is an `'transient_display_data'` message.
+   */
+  export function isTransientDisplayDataMsg(
+    msg: IMessage
+  ): msg is ITransientDisplayDataMsg {
+    return msg.header.msg_type === 'transient_display_data';
+  }
+
+  /**
+   * A `'transient_display_data'` message on the `'iopub'` channel.
+   */
+  export interface ITransientDisplayDataMsg extends IDisplayDataMsg {
+    content: IDisplayDataMsg['content'];
+  }
+
   /**
```

Comment:
The definition of this message type is technically not needed for this PR to work.